lib/filterx/func-unset-empties.c

```diff
@@ -402,6 +402,7 @@ _handle_target_object(FilterXFunctionUnsetEmpties *self, FilterXObject *target,
     }
   else if (filterx_object_is_type(target, &FILTERX_TYPE_NAME(string)))
     {
+      gsize len;
```

Comment:
isn't this triggering a shadowed name issue?

Reply:
it does, that's why the compilation failed.

Reply:
Indeed. I've had too much Rust it seems.

Reply:
We should probably unify this, gsize would be a good candidate to represents length.

Reply:
I agree, filterx_object_len() could return gsize, but I am merging this as is.